### DISCUSSION TITLE
Something that has nothing to do with 1T

### DISCUSSION THREAD

**Discussion by rosgr100** [Marekkon5/onetagger#304] [General]
So, @Marekkon5 @bascurtiz your work is amazing, it saved me so much time and made my workflow so much more efficient.
This is an amazing tools -i would actually call it a present- for any Dj out there. 
Seeing what you can achieve, I wonder if you have any interest to take a look on the -nowadays- open source code of the -once before- evolutionary and now almost dead, Rekordbuddy?
The person who got responsible for the code is open to help, he is actually looking for some.
The revival of rekordbuddy would be amazing for the DJ community, but again I understand that this requires a lot of time which is not always available.

You can find the code on codeberg [here](https://codeberg.org/DidierMalenfant/RekordBuddy)

Thank you!

**Comment by bascurtiz** [comment]
I mean, from the basic principles of One Tagger there is no need for RekordBuddy...
Which main purpose is making sure one db gets copied over properly to the other db/app
Since we have all written within the metadata.

So can you clarify why you think it still useful one way or another? Or maybe outside the scope of One Tagger?

**Comment by rosgr100** [comment]
Of course, it has nothing to do with One Tagger's purpose. 
The main function of rekordbuddy was to synchronize collections (cues included) and playlists between the major dj apps. Nowadays there a few alternatives out there ,but especially for windows their -subscription- pricing is outrageous. For Mac there are the tools by atgr (djcu) etc, but windows alternatives are somehow limited and expensive.
